sshd-common/src/main/java/org/apache/sshd/common/config/keys/OpenSshCertificate.java

```diff
@@ -35,6 +36,10 @@
     int SSH_CERT_TYPE_USER = 1;
     int SSH_CERT_TYPE_HOST = 2;
 
+    long VALID_AFTER_FOREVER_EPOCH = 0L;
+    // note that this is the max value of a uint64 (java.lang.Long.MAX_VALUE is the max of a *signed* int64)
+    long VALID_BEFORE_FOREVER_EPOCH = 0xffffffffffffffffL;
```

Comment:
Either write this as ~0L, or use underscores to make this more readable: 0xffff_ffff_ffff_ffffL is much easier to read, and one immediately sees that there are indeed 16 'f's.